lib/improver/nbhood.py

```diff
@@ -169,9 +172,13 @@ def get_neighbourhood_width_in_grid_cells(
 
 class SquareNeighbourhood(object):
 
+    """
+    Methods for use in application of a square neighbourhood.
+    """
+
     def __init__(self):
```

Comment:
Corrected so that the interfaces of the `CircularNeighbourhood` and `SquareNeighbourhood` class are the same.